core/state_transition.go

```diff
@@ -269,8 +269,18 @@ func (st *StateTransition) buyGas() error {
 }
 
 func (st *StateTransition) preCheck() error {
+	// Only check transactions that are not fake
+	if !st.msg.IsFake() {
+		// Make sure the sender is an EOA
+		code := st.state.GetCode(st.msg.From())
```

Comment:
Will this change result in the code for the sender account being included in the trace?

Reply:
yes. it will be loaded if the sender account has code: https://github.com/scroll-tech/go-ethereum/blob/d3189603df19793e69d83da31c4c613f20a16806/core/state/statedb.go#L299-L304